client/src/pages/Profile/Profile.tsx

```diff
@@ -1,18 +1,29 @@
-import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
+import ProfileSideBar from './ProfileSideBar/ProfileSideBar';
 import useStyles from './useStyles';
 
-const Profile = (): JSX.Element => {
-  const classes = useStyles();
+interface Props {
+  children?: React.ReactNode;
+}
 
+const Profile = ({ children }: Props): JSX.Element => {
+  const classes = useStyles();
   return (
-    <Grid className={classes.profile}>
-      <Box>
-        <Typography component="h5" variant="h5" align="center">
-          Profile Page Template
-        </Typography>
-      </Box>
+    <Grid container className={classes.profile}>
+      <ProfileSideBar />
+      {children ? (
```

Comment:
As opposed to displaying the children here, you can extract the 'path' name from the url and conditionally render the component which matches the path name.

Reply:
Applied.
I extracted the ```pathname``` from ```useLocation()``` and rendered components conditionally with ```switch``` statement. It seems to work fine as I intended. But, I am not sure whether if I fixed it nicely or not.